packages/contracts/src/predeploys.ts

```diff
@@ -3,7 +3,7 @@ export const predeploys = {
   OVM_L1MessageSender: '0x4200000000000000000000000000000000000001',
   OVM_DeployerWhitelist: '0x4200000000000000000000000000000000000002',
   OVM_ECDSAContractAccount: '0x4200000000000000000000000000000000000003',
-  OVM_ProxySequencerEntrypoint: '0x4200000000000000000000000000000000000004',
+  //OVM_ProxySequencerEntrypoint: '0x4200000000000000000000000000000000000004', // removed
```

Comment:
Yeah I think this makes sense

Reply:
Why comment it out instead of removing it?

Reply:
Ah I thought it might be weird if the addresses go from `0x42....03` to `0x42....05` without explanation. Someone might think there's a missing contract there. I figured we could leave this here for posterity until we figure out what to do about predeploys we're no longer using.

Reply:
I think it would be better to address with a comment on top of the `predeploys` object instead of commenting out the line. That will give room to explain why it was removed

Reply:
Comment added.